src/util/json.h

```diff
@@ -11,5 +12,26 @@
 // most notably to std::string.
 #define JSON_USE_IMPLICIT_CONVERSIONS 0
 #include <nlohmann/json.hpp>
+#include <optional>
+
+// Adding third party type support.
```

Comment:
```suggestion
// Add support for serializing `std::optional` using `nlohmann::json`
```

Reply:
Done in [866450c](https://github.com/ad-freiburg/qlever/pull/860/commits/866450c81a52f7c90ff40642b2641eeded4b8210), but I rephrased `Add` to `Added`, because with `Add` it sounds like something that has yet to be done.
For example: "Add water to the mix." doesn't mean, that water is already in the mix, but that you should add it.